app/models/theme_field.rb

```diff
@@ -138,6 +138,8 @@ def process_extra_js(content)
         js_compiler.append_module(content, filename)
       when "hbs"
         js_compiler.append_ember_template(filename.sub("discourse/templates/", ""), content)
+      when "hbr"
```

Comment:
You can combine the "`when`" instead of duplicating the code 😉 

```suggestion
      when "hbr", "raw.hbs"
        js_compiler.append_raw_template(filename.sub("discourse/templates/", ""), content)
```